### ISSUE TITLE
LED with label not rendered correctly

### ISSUE THREAD

**Issue by georgweiss** [ControlSystemStudio/phoebus#2199]
A user has created an OPI heavily using the LED widget together with a label. When initially loaded some of the widgets do not render the label text and user will need to reload display once or twice before all labels are rendered as expected.

Not calling `pane.setManaged(false);` in `createJFXNode`, or calling `jfx_node.layout()` and the end of `updateChanges` in `BaseLEDRepresentation` seems to solve the issue.

@kasemir, any thoughts?

**Comment by kasemir** [comment]
`setManaged(false)` was added to several widgets because it dramatically improved rendering speed.
Details might by now have changed, and we have reverted back to using managed mode on several widgets because it simply became to hard to find the correct location for calling `layout`, and without `layout`, there were rendering errors.

Still, as step 1 I'd try adding the appropriate `layout` call, and if that works out, fine.
Fallback would be to revert all the way back to using managed mode, but if possible I'd avoid that because especially with many "simple" widgets managed mode resulted in endless recursions inside the JFX rendering code.
